init/Makefile

```diff
@@ -1,11 +1,25 @@
-CC       = gcc
-C_FLAGS := -Wall -Wextra -Werror -O2
+
+ARCH_aarch64	               = aarch64
+C_FLAGS 	              := -Wall -Wextra -Werror -O2
```

Comment:
These 2 (C_FLAGS and CC) should be aligned with the other ones; if not already, and it's only viewing like this on GitHub.

Reply:
Looks ok now.